src/hooks/useToken.js

```diff
@@ -23,24 +23,32 @@ export function useTokenBalanceOf(tokenAddress, account) {
 
 export function useTokenData(tokenAddress) {
   const [tokenData, setTokenData] = useState({ decimals: 18, symbol: '' })
-  const [loading, setLoading] = useState(true)
+  const [loading, setLoading] = useState(false)
 
   const mounted = useMounted()
   const tokenContract = useContractReadOnly(tokenAddress, tokenAbi)
 
   useEffect(() => {
     const fetchTokenData = async () => {
+      if (tokenAddress) {
+        setLoading(true)
+      }
+      const name = await tokenContract.name()
       const decimals = await tokenContract.decimals()
       const symbol = await tokenContract.symbol()
```

Comment:
If an address is not a token address, with a name and symbol, it crashes. Need some guidance to treat that exception.

Reply:
With the new pushes this bug is solved. As long as I am concerned everything works correctly.